src/ORM/Query.php

```diff
@@ -386,7 +395,12 @@ public function contain($associations = null, $override = false)
             return $loader->contain();
         }
 
-        $result = $loader->contain($associations);
```

Comment:
I'm not sure what you mean :)

The EagerLoader is what does the work so it needs to have the new signature first, then Query::contain needs to just support it.